monkey/monkey_island/cc/ui/src/styles/pages/ConfigurationPage.scss

```diff
@@ -49,10 +49,6 @@
   font-size: 1.2em;
 }
 
-.config-field-hidden {
-  display: none;
-}
-
```

Comment:
If nothing throws errors, I would leave this be (or comment it out) for now just so that it's easy if we ever have to use this again. No big deal, though. Your choice.